components/schema-migrator/Dockerfile

```
@@ -1,28 +1,23 @@
 FROM golang:1.18.3-alpine3.15 AS build
 
-WORKDIR /migrate
+COPY go.mod /program/go.mod
+COPY main.go /program/main.go
 
-RUN apk --no-cache add --update git make
-RUN git clone https://github.com/golang-migrate/migrate.git .
+WORKDIR /program
+RUN go mod tidy && go mod vendor && \
+    CGO_ENABLED=0 go build -o /bin/program /program/main.go
```

Stripping would it make it even smaller :-) 
```suggestion
    CGO_ENABLED=0 go build -ldflags="-s -w" -o /bin/program /program/main.go
```

---

But then you lose convenience of continuous profiling tools based on ebpf and similar technologies..